src/pages/courses/_layouts/CourseLayout.astro

```diff
@@ -201,6 +291,32 @@ const structuredData = JSON.stringify({
         </Features>
       )
     }
+    {
+      prerequisites && (
+        <section>
```

Comment:
I just tested this on multiple devices.

Adding a section with the roadmap tends to break the scrolling flow because the graph fills the screen and once I'm inside, I can only zoom in or out once I scroll. But then again the interaction with the graph is natural once it's in front of you, so it's supposed to work that way. There's nothing wrong with the implementation. I think the original design/placement could be changed.

I'm thinking we could move the Roadmap in a modal dialog displayable by a button, so that the reader can check it out if they're interested? I'm thinking of adding a button next to "enroll now" in the hero section, e.g. "show roadmap", and it could open a modal dialog where we have this graph.

Any other ideas or suggestions?